assets/themes/campersites/campersites.json

```diff
@@ -0,0 +1,718 @@
+              "if": {
+                "and": [
+                  "sanitary_dump_station:grey_water=yes"
+                ]
+              },
+              "then": {
+                "en": "You can dispose of grey water here"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "sanitary_dump_station:grey_water=no"
+                ]
+              },
+              "then": {
+                "en": "You cannot dispose of gray water here"
+              }
+            }
+          ]
+        },
+        {
+          "render": {
+            "en": "chemical toilet"
+          },
+          "question": {
+            "en": "Can you dispose of chemical toilet waste here?"
+          },
+          "freeform": {
+            "key": "sanitary_dump_station:chemical_toilet"
+          },
+          "mappings": [
+            {
+              "if": {
+                "and": [
+                  "sanitary_dump_station:chemical_toilet=yes"
+                ]
+              },
+              "then": {
+                "en": "You can dispose of chemical toilet waste here"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "sanitary_dump_station:chemical_toilet=no"
+                ]
+              },
+              "then": {
+                "en": "You cannot dispose of chemical toilet waste here"
+              }
+            }
+          ]
+        },
+        {
+          "render": {
+            "en": "access tag"
+          },
+          "question": {
+            "en": "Who can use this dump station?"
+          },
+          "freeform": {
+            "key": "access"
+          },
+          "mappings": [
+            {
+              "if": {
+                "and": [
+                  "access=network"
+                ]
+              },
+              "then": {
+                "en": "You need a network key/code to use this"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "access=customers"
+                ]
+              },
+              "then": {
+                "en": "You need to be a customer of camping/campersite to use this place"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "access=public"
+                ]
+              },
+              "then": {
+                "en": "Anyone can use this dump station"
+              },
+              "hideInAnswer": false
+            },
+            {
+              "if": {
+                "and": [
+                  "access=yes"
+                ]
+              },
+              "then": {
+                "en": "Anyone can use this dump station"
+              },
+              "hideInAnswer": true
+            }
+          ]
+        },
+        {
+          "render": {
+            "en": "This place is operated by {operator}"
```

Comment:
The common questions for all layers can be moved to `roamingRenderings` of the the theme - these are tagRenderings that are added to every object in every layer, this avoids repetition and makes fixes and translations easier.

(If you want to hide a tagRendering for a single layer, this can be done by the condition on the tagrendering)